src/cli/commands/bundle.ts

```diff
@@ -0,0 +1,44 @@
+import { Command, Option } from "clipanion"
+import { bundle } from "src/bundle/bundle"
+import fs from "node:fs/promises"
+import prettyBytes from "pretty-bytes"
+
+export class BundleCommand extends Command {
+  static paths = [[`bundle`]]
+
+  static usage = Command.Usage({
```

Comment:
possibly, by default this is outputting a file that can either be used for an embedded/module service or you can create your own 3 loc entrypoint importing the bundle

maybe eventually an option to embed the adapter would be nice too

Reply:
oh right, I'm getting things mixed up, this just exports the "route bundle"

I guess I'm a little confused because `bundle` takes a "bundledAdapter" option which can bundle the wintercg adapter....is that even needed anymore?

Reply:
yeah, I'm adding a todo comment about that option in my next PR
we do need it but it's only used internally for dev and test fixtures so 🤷